index.html

```diff
@@ -9,76 +9,259 @@
+        <div id="rectang"></div>
 
-        <normal_btn id="normal_btn">
-            <icons id="icons">
+        <div id="normal_btn">
+            <div id="icons">
                 <union1 id="union1">
-                    <vectorx id="vectorx">
-                    <vectorx>
-                    <rec id="rec">
-                        <img src="./assets/Icon - Mail.png" alt="">
-                    </rec>
-                </union1>
-            </icons>
-        </normal_btn>
-
-        <nav id="nav">
-            <img src="./assets/Enabled.svg" alt="">
-        </nav>
+                    <div id="vectorx">
+                    <div>
+                    <div id="rec">
+                        <img src="./assets/Icon - Mail.png" alt="Messages">
+                    </div>
+                  </div>
+                </div>
+          </div>
+
+        <div id="nav">
+            <img src="./assets/Enabled.svg" alt="See project">
+        </div>
     </header>
 
 
     <section id="section">
         <h1 id="h1"><p>Hey There, I'm Alex.<p></h1>
         <h2 id="h2"><p>I'm a Software Developer.</p></h2>
-        <description id="description"><p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></description>
+        <div id="description"><p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></div>
 
         
     <ul id="ul">
         <li>
-            <a href="https://github.com"> <img src="./assets/github.svg" alt=""></a>
+            <a href="https://github.com"> <img src="./assets/github.svg" alt="Git hub"></a>
         </li>
         <li>
-            <a href="https://linkedin.com"> <img src="./assets/Linkedin icon.svg" alt=""></a>
+            <a href="https://linkedin.com"> <img src="./assets/Linkedin icon.svg" alt="Linkedin"></a>
         </li>
         <li>
-            <a href="https://angellist.com"> <img src="./assets/angel list.svg" alt=""></a>
+            <a href="https://angellist.com"> <img src="./assets/angel list.svg" alt="Angel list"></a>
         </li>
         <li>
-            <a href.="https://twitter.com"> <img src="./assets/twitter.svg" alt=""></a>
+            <a href.="https://twitter.com"> <img src="./assets/twitter.svg" alt="Twitter"></a>
         </li>
         <li>
-            <a href="https://medium.com"> <img src="./assets/medium.svg" alt=""></a>
+            <a href="https://medium.com"> <img src="./assets/medium.svg" alt="Medium"></a>
         </li>
     </ul>
     </section>
 
         
-        <illustration id="illustration">
-        </illustration>
+        <div id="illustration">
+        </div>
+
+    <div id="base1"></div>
+
+
+    <recentwork id="recentwork"><p id="recentwork_text">My Recent Works</p>
```

Comment:
Kindly, in previous reviews I told you just to use valid HTML labels, and turn them into correct or divs, I attached documentation about HTML and its correct uses. (Check all your code to remove them all) https://www.tutorialspoint.com/html5/html5_tags.htm